core_utils/article.py

```diff
@@ -4,6 +4,9 @@
 import json
```

Comment:
you can not change article.py class. It is given you as module - just use it as is

Reply:
Got it.